volatility3/framework/layers/intel.py

```diff
@@ -264,12 +269,41 @@ def _mapping(self,
                 length -= chunk_size
                 offset += chunk_size
 
+            if offset > self.maximum_address:
```

Comment:
This is now only testing whether the address is greater than the largest possible register value.

Reply:
This check is there to prevent wrap around.